packages/firebase_analytics/firebase_analytics_web/lib/firebase_analytics_web.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:firebase/firebase.dart' as firebase;
+import 'package:firebase_analytics_platform_interface/firebase_analytics_platform_interface.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:meta/meta.dart';
+
+/// Web implementation for [FirebaseAnalyticsPlatform]
+class FirebaseAnalyticsWeb extends FirebaseAnalyticsPlatform {
+  /// Instance of Analytics from the web plugin.
+  final firebase.Analytics _analytics;
+
+  /// A constructor that allows tests to override the firebase.Analytics object.
+  FirebaseAnalyticsWeb({@visibleForTesting firebase.Analytics analytics})
+      : _analytics = analytics ?? firebase.analytics();
+
+  /// Called by PluginRegistry to register this plugin for Flutter Web
+  static void registerWith(Registrar registrar) {
+    FirebaseAnalyticsPlatform.instance = FirebaseAnalyticsWeb();
+  }
+
+  @override
+  Future<void> logEvent({
+    String name,
+    Map<String, dynamic> parameters,
+  }) {
+    _analytics.logEvent(name, parameters);
+    return Future<void>.value();
```

Comment:
If you mark this logEvent as async, you don't have to return an empty future (IIRC you don't have to return anything!)

(Same for all the return Future<void>.value() of this class!)

Reply:
Aweeeeesome! Thanks :)!